src/lib/smsClient.js

```diff
@@ -1397,6 +1397,109 @@ const getLogForwarding = async (organizationCode, projectId, workspaceId, meshId
 	}
 };
 
+/**
+ * Deletes the log forwarding configuration for a given mesh.
+ *
+ * @param {string} organizationCode - The IMS org code
+ * @param {string} projectId - The project ID
+ * @param {string} workspaceId - The workspace ID
+ * @param {string} meshId - The mesh ID
+ */
+const deleteLogForwarding = async (organizationCode, projectId, workspaceId, meshId) => {
+	const { accessToken } = await getDevConsoleConfig();
+	const config = {
+		method: 'DELETE',
+		url: `${SMS_BASE_URL}/organizations/${organizationCode}/projects/${projectId}/workspaces/${workspaceId}/meshes/${meshId}/log/forwarding`,
+		headers: {
+			'Authorization': `Bearer ${accessToken}`,
+			'x-request-id': global.requestId,
+			'x-api-key': SMS_API_KEY,
+		},
+	};
+
+	logger.info(
+		'Initiating DELETE %s',
+		`${SMS_BASE_URL}/organizations/${organizationCode}/projects/${projectId}/workspaces/${workspaceId}/meshes/${meshId}/log/forwarding`,
+	);
+
+	try {
+		const response = await axios(config);
+
+		logger.info('Response from DELETE %s', response.status);
+
+		if (response && response?.status === 204) {
+			return response;
+		} else {
+			logger.error(
+				`Something went wrong: ${objToString(
```

Comment:
Not sure if I like this "Something went wrong". I think just returning "Unable to delete log forwarding config" and the error message should suffice